### PR TITLE
[SMAGENT-3940] Implement KSM Ingress metrics

### PR DESCRIPTION
**Pull request by fremmi**

### CODE REVIEW

**Review by irozzo-1A — APPROVED** [review]
LGTM

File: agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml [hunk 1]
```diff
@@ -55,6 +55,7 @@ rules:
   - networking.k8s.io
   resources:
   - networkpolicies
+  - ingresses
```
Comment (balinaveen) [inline agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml:58]: This will require a change in our helm chart too, right? cc @achandras

Comment (irozzo-1A) [reply agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml:58]: yep, that's true. We should also clean-up `ingresses` from the `extensions` api group that is no longer needed

Comment (irozzo-1A) [reply agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml:58]: https://github.com/draios/sysdig-cloud-scripts/blob/1de3393a0c2396c2c3558c5a3681a687a46bc181/agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml#L68
cc @fremmi

**Review by irozzo-1A — CHANGES_REQUESTED** [review]
@fremmi We should also align the openshift manifest:
https://github.com/draios/sysdig-cloud-scripts/blob/1de3393a0c2396c2c3558c5a3681a687a46bc181/agent_deploy/openshift/sysdig-agent-clusterrole.yaml

File: agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml (hunk 1 above)
Comment (achandras) [reply agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml:58]: @irozzo-1A is there any downside to just leaving the extensions apigroup for anyone on really old kubernetes? I'm mainly thinking of openshift 3 which we still test, which matches k8s v1.11, I believe

Comment (irozzo-1A) [reply agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml:58]: no downside, but it's not used anywhere even by old agent versions so I think that makes sense to just remove it

**Review by irozzo-1A — APPROVED** [review]
LGTM

**Review by federico-sysdig — APPROVED** [review]

**Pull request merged**